src/language-js/utils/index.js

```diff
@@ -475,6 +459,35 @@ function isUnitTestSetUp(node) {
   );
 }
 
+function isTestCallCallee(node) {
+  const patterns = [
```

Comment:
Should we move this array to outer scope?

Reply:
I followed [this way](https://github.com/prettier/prettier/blob/b188c905cfaeb238a122b4a95c230da83f2f3226/src/language-js/utils/index.js#L470).

And I thought the smaller the scope of the variable, the better.

However, moving this array to outer scope may be more good performance (?) due to using less memory.

 I want follow your decision.

Reply:
> moving this array to outer scope may be more good performance (?) due to using less memory.

I guess so.  `CallExpression` is so common in ESTree, Let's move it out then.

Reply:
I moved .

Please let me know if there are any bad points.